### PR TITLE
add uniform op doc

### PR DESCRIPTION
**Pull request by pangyoki**
Add uniform op under paddle/tensor/random.py. Including cn and en doc.

## 中文

屏幕快照 2020-08-21 下午3 47 05
屏幕快照 2020-08-21 下午3 48 23

## 英文

屏幕快照 2020-08-21 下午3 49 29
屏幕快照 2020-08-21 下午3 50 32

### CODE REVIEW

File: doc/fluid/api_cn/tensor_cn/uniform_cn.rst [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+    示例1:
+             给定：
+                 shape=[1,2]
+             则输出为：
+                 result=[[0.8505902, 0.8397286]]
+
+参数：
+    - **shape** (list|tuple|Tensor) - 生成的随机Tensor的形状。如果 ``shape`` 是list、tuple，则其中的元素可以是int，或者是形状为[1]且数据类型为int32、int64的Tensor。如果 ``shape`` 是Tensor，则是数据类型为int32、int64的1-D Tensor。
+    - **dtype** (str|np.dtype|core.VarDesc.VarType, 可选) - 输出Tensor的数据类型，支持float32、float64。默认值为float32。
```
Comment (jzhang533) [inline doc/fluid/api_cn/tensor_cn/uniform_cn.rst]: no core.VarDesc.VarType

File: doc/fluid/api_cn/tensor_cn/uniform_cn.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    shape = np.array([2, 3])
+    shape_tensor = paddle.to_tensor(shape)
+
+    result_3 = paddle.uniform(shape_tensor)
+    # if shape_tensor's value is [2, 3]
+    # result_3 is:
+    # [[-0.8517412,  -0.4006908,   0.2551912 ],
+    #  [ 0.3364414,   0.36278176, -0.16085452]]
+
+    paddle.enable_static()
```
Comment (jzhang533) [inline doc/fluid/api_cn/tensor_cn/uniform_cn.rst]: 不需要这一行.

Comment (pangyoki) [reply doc/fluid/api_cn/tensor_cn/uniform_cn.rst]: Done

File: doc/fluid/api_cn/tensor_cn/uniform_cn.rst (hunk 1 above)
Comment (pangyoki) [reply doc/fluid/api_cn/tensor_cn/uniform_cn.rst]: Done

**Review by jzhang533 — APPROVED** [review]
lgtm

**Review by zhiqiu — APPROVED** [review]
LGTM for doc.
A small question is that when `dtype` is None, should we use global dtype, which is managed by `get_default_dtype` and `set_default_dtype`

**Pull request merged**